src/base/memory_monitor_sr.cpp

```diff
@@ -142,14 +150,16 @@ namespace cubperf
       memory_monitor &operator = (const memory_monitor &) = delete;
       memory_monitor &operator = (memory_monitor &&) = delete;
 
-      int add_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID stat_id, uint64_t size, bool expand);
-      int sub_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID stat_id, uint64_t size);
-      int resize_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID stat_id, uint64_t old_size, uint64_t new_size);
-      int move_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID src, MMON_STAT_ID dest, uint64_t size);
+      void add_stat (MMON_STAT_ID stat_id, int64_t size);
+      void add_tran_stat (THREAD_ENTRY *thread_p, int64_t size);
+      void resize_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID stat_id, uint64_t old_size, uint64_t new_size);
+      void move_stat (THREAD_ENTRY *thread_p, MMON_STAT_ID src, MMON_STAT_ID dest, uint64_t size);
+      void end_init_phase ();
       int aggregate_module_info (MMON_MODULE_INFO *info, int module_index);
       int aggregate_tran_info (MMON_TRAN_INFO *info, int tran_count);
-      inline int get_comp_idx_from_comp_idx_map (int comp_idx_map_val);
-      inline int get_subcomp_idx_from_comp_idx_map (int comp_idx_map_val);
+
+    private:
+      inline int get_module_idx (MMON_STAT_ID stat_id);
```

Comment:
```suggestion
      inline int get_module_idx (MMON_STAT_ID stat_id) const;
```
As many const as possible. It represents immutability explicitly and prevents mistakes. I think many of methods can be const including the `aggregate_*` functions.

Reply:
Done.